packages/create-rspress/template/tsconfig.json

```diff
@@ -1,8 +1,19 @@
 {
   "compilerOptions": {
+    "target": "ES2020",
+    "lib": ["DOM", "ES2020"],
+    "module": "ESNext",
     "jsx": "react-jsx",
-    "esModuleInterop": true
+    "noEmit": true,
+    "strict": true,
+    "skipLibCheck": true,
+    "isolatedModules": true,
+    "resolveJsonModule": true,
+    "moduleResolution": "bundler",
+    "useDefineForClassFields": true,
+    "allowImportingTsExtensions": true
   },
+  "include": ["docs", "src"],
```

Comment:
should include the `theme` folder, as Rspress recommend users to use this folder to custom theme.

https://rspress.dev/guide/advanced/custom-theme#1-basic-structure